taylor-francis/taylor-francis-proposal-2020-07.md

```diff
@@ -0,0 +1,308 @@
+# Building Research Software
+
+-   Damien Irving
+-   Kate Hertweck
+-   Luke Johnston
+-   Sara Mahallati
+-   Joel Ostblom
+-   Charlotte Wickham
+-   Greg Wilson
+
+## 1. Aims
+
+Software is now as essential to research as telescopes, test tubes, and
+reference libraries, which means that researchers need need to know how to
+build, check, use, and share programs.  However, most introductions to
+programming focus on developing commercial applications, not on exploring
+problems whose answers aren't yet known.  Our goal is show readers how to do
+that, both on their own and as part of a team.
+
+We believe every researcher should know how to write short programs that clean
+and analyze data in a reproducible way, and how to use version control to keep
```

Comment:
Maybe just say "analyze data in a reproducible way" and leave out the reference to data cleaning, because that's not really a focus of this book (I'm assuming it will be in the novice book).

Reply:
:+1: